lighthouse-extension/gulpfile.js

```diff
@@ -178,6 +207,14 @@ gulp.task('watch', ['lint', 'browserify', 'html'], () => {
     'app/src/**/*.js',
     '../lighthouse-core/**/*.js'
   ], ['browserify']);
+
+  gulp.watch([
+    '../lighthouse-core/**/*.html'
```

Comment:
So they can't be combined? Also can we utilize `config.report` and `config.partials` for the globs?